CHANGES.txt

```diff
@@ -12,6 +12,10 @@ Bugfixes
 * Fix margins of paragraphs at the end of sections (Issue #3704)
 * Ignore ``.DS_Store`` files in listing indexes (Issue #3698)
 * Fix baguetteBox.js invoking in the base theme (Issue #3687)
+* Fix development (preview) server `nikola auto`
+  for non-root SITE_URL, in particular when URL_TYPE is full_path.
+  (This does not yet fix `nikola serve`,
+  see https://github.com/getnikola/nikola/issues/3726 .)
```

Comment:
I don’t think it’s useful to mention another issue here. Please do add `(Issue #3715)` to the line.

```suggestion
  (Issue #3715)
```